internal/consts/consts.go

```diff
@@ -79,14 +79,22 @@ const (
 	FieldAuthLoginOIDC            = "auth_login_oidc"
 	FieldAuthLoginJWT             = "auth_login_jwt"
 	FieldAuthLoginAzure           = "auth_login_azure"
-	FieldIdentity                 = "identity"
-	FieldSignature                = "signature"
-	FieldPKCS7                    = "pkcs7"
-	FieldNonce                    = "nonce"
 	FieldIAMHttpRequestMethod     = "iam_http_request_method"
 	FieldIAMHttpRequestURL        = "iam_http_request_url"
 	FieldIAMHttpRequestBody       = "iam_http_request_body"
 	FieldIAMHttpRequestHeaders    = "iam_http_request_headers"
+	FieldAWSAccessKeyID           = "aws_access_key_id"
```

Comment:
nit: I wonder if we should now rename the pre-existing constants defined here to be more generic. Like `FieldAccessKey` and `FieldSecretKey` https://github.com/hashicorp/terraform-provider-vault/blob/main/internal/consts/consts.go#L48

Reply:
Funny you should say that, I did just that for the Login MFA work.

Reply:
Nice!